ui-modules/blueprint-composer/app/components/catalog-selector/catalog-selector.directive.js

```diff
@@ -30,15 +47,43 @@ export function catalogSelectorDirective() {
         scope: {
             family: '<',
             onSelect: '&',
-            itemsPerPage: '<',
+            rowsPerPage: '<',  // if unset then fill
```

Comment:
If this parameter is option, it should have a `?` at the end, like the ones below

Reply:
This parameter is optional, it should have a `?` at the end, like the ones below.